_posts/2000-01-03-details.md

```diff
@@ -35,4 +35,4 @@ If the installation doesn't work for you, check out [common errors documentation
 
 ### Simply join our channel at [Gitter](https://coala.io/chat)! We'd love to speak to you!
 
-### Look at [our tutorial](https://coala.readthedocs.io/en/latest/Users/Tutorials/Tutorial.html) for a deeper introduction.
+### Look at [our tutorial](https://coala.io/tutorial) for a deeper introduction.
```

Comment:
@chauffer @sils how is the redirection being done?
Where do I get that https://coala.io/tutorial is redirecting to https://docs.coala.io/en/latest/Users/Tutorial.html ?